README.md

```diff
@@ -20,7 +20,7 @@ table = Shapefile.Table(path)
 geoms = Shapefile.shapes(table)
 
 # whole columns can be retrieved by their name
-table.Descriptio  # => Union{String, Missing}["Square with triangle missing", "Smaller triangle", missing]
+table.Description  # => Union{String, Missing}["Square with triangle missing", "Smaller triangle", missing]
```

Comment:
It really is just `Descriptio`, just open the dbf in a plain text editor.
http://switchfromshapefile.org/#10characters
:(

Reply:
The horror